src/components/screens/SchoolScheduleCreation.js

```diff
@@ -55,38 +53,48 @@ class SchoolScheduleCreation extends React.Component {
 						else this.props.navigation.pop();
 					});
 				})
-				.catch(err => console.log('eerr', err));
+				.catch(err => console.log('error', err));
 		}
 	}
 
-	progressValue() {
-		return setInterval(() => {
-			if (this.state.width <= 1) {
-				let width  = this.state.width + 0.01;
-				this.setState({width});
-			}
-		},10);
-	}
-
 	render() {
 		return(
 			<LinearGradient style={styles.container} colors={gradientColors}>
-				<ImageBackground style={styles.container} source={require('../../assets/img/loginScreen/backPattern.png')} resizeMode="repeat">
-					<StatusBar translucent={true} backgroundColor={'rgba(0, 0, 0, 0.4)'} />
+				<ImageBackground style={styles.container} 
+					source={require('../../assets/img/loginScreen/backPattern.png')} 
+					resizeMode="repeat"
+				>
+					<StatusBar translucent={true} 
+						backgroundColor={'rgba(0, 0, 0, 0.4)'} 
+					/>
```

Comment:
Put the closing bracket on the same line of the last prop (we forgot to add it in the coding convention)